Libraries/Image/RCTImageView.h

```diff
@@ -38,15 +32,6 @@ typedef NS_ENUM(NSInteger, UIImageRenderingMode) {
 @property (nonatomic, assign) RCTResizeMode resizeMode;
 
 #if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
-/**
- * macOS Properties
- */
-@property (nonatomic, copy) RCTDirectEventBlock onDoubleClick;
-@property (nonatomic, copy) RCTDirectEventBlock onClick;
-@property (nonatomic, copy) RCTDirectEventBlock onMouseEnter;
-@property (nonatomic, copy) RCTDirectEventBlock onMouseLeave;
-@property (nonatomic, copy) RCTDirectEventBlock onDragEnter;
-@property (nonatomic, copy) RCTDirectEventBlock onDragLeave;
-@property (nonatomic, copy) RCTDirectEventBlock onDrop;
+@property (nonatomic, strong) RCTUIColor *tintColor;
```

Comment:
Are we matching another property with strong? Or should we just be using copy since NSColor conforms to NSCopying

Reply:
It was mimicking the `tintColor` property of `UIView`, but you're right, it should really be the same as the `contentTintColor` of `NSImageView` and other NSColor properties in AppKit.    Also, RCTImageview doesn't need an ivar, it should just return the underlying NSImageView's contentTintColor.